### PR TITLE
Added standings page UI

### PR DESCRIPTION
**Pull request by Yash-jar**
added a standing board with a filter widget

### CODE REVIEW

**Review by Hareesh-Nandigrama — CHANGES_REQUESTED** [review]
image
Yeh blue line stays after selecting
image
This is its corresponding error, try to resolve it

File: packages/scoreboard/lib/src/widgets/scoreboard/standingboard.dart [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  @override
+  State<StandingBoard> createState() => _StandingBoardState();
+}
+
+class _StandingBoardState extends State<StandingBoard> {
+  var _list = [0, 1, 2];
+  var _images = [
+    'packages/scoreboard/assets/images_standingboard/GoldMedal.png',
+    'packages/scoreboard/assets/images_standingboard/SilverMedal.png',
+    'packages/scoreboard/assets/images_standingboard/BronzeMedal.png',
```
Comment (vrrao01) [inline packages/scoreboard/lib/src/widgets/scoreboard/standingboard.dart:22]: @Yash-jar Use Icons.workspace_premium for the medals. Delete the png files you've added.

File: packages/scoreboard/pubspec.yaml [hunk 2]
```diff
@@ -22,6 +22,7 @@ dependencies:
   dio: ^4.0.6
   intl: ^0.18.0
   dropdown_textfield: ^1.0.7
+  font_awesome_flutter: ^10.3.0
```
Comment (vrrao01) [inline packages/scoreboard/pubspec.yaml:25]: What are we using `font_awesome_flutter` for?  @Yash-jar

**Review by vrrao01 — DISMISSED** [review]

**Review by Hareesh-Nandigrama — APPROVED** [review]

**Review by Hareesh-Nandigrama — APPROVED** [review]

File: packages/scoreboard/pubspec.yaml (hunk 2 above)
Comment (Yash-jar) [reply packages/scoreboard/pubspec.yaml]: removed it

File: packages/scoreboard/lib/src/widgets/scoreboard/standingboard.dart (hunk 1 above)
Comment (Yash-jar) [reply packages/scoreboard/lib/src/widgets/scoreboard/standingboard.dart]: deleted them

**Pull request merged**